include/SFML/Graphics/Texture.h

```diff
@@ -68,6 +68,31 @@ CSFML_GRAPHICS_API sfTexture* sfTexture_create(unsigned int width, unsigned int
 ////////////////////////////////////////////////////////////
 CSFML_GRAPHICS_API sfTexture* sfTexture_createFromFile(const char* filename, const sfIntRect* area);
 
+////////////////////////////////////////////////////////////
+/// \brief Create a texture configured for srgb from a file
+/// When providing texture data from an image file or memory, it can
```

Comment:
I think there should be an empty comment line between brief and the extended text